templates/template_helpers.go

```diff
@@ -55,6 +55,12 @@ func CreateTemplateHelpers(templatePath string) template.FuncMap {
 		"snakeCase": snakeCase,
 		"camelCase": camelCase,
 		"camelCaseLower": camelCaseLower,
+		"plus": wrapFloatFloatToFloatFunction(func(arg1 float64, arg2 float64) float64 { return arg1 + arg2 }),
```

Comment:
Is `wrapFloatFloatToFloatFunction` a typo? Looks like this should be `wrapFloatToFloatFunction`.